kafka_consumer/tests/test_unit.py

```diff
@@ -74,18 +70,20 @@ def test_tls_config_ok(check, kafka_instance_tls):
     ],
 )
 @pytest.mark.skipif(not LEGACY_CLIENT, reason='not implemented yet with confluent-kafka')
-def test_oauth_config(sasl_oauth_token_provider, check, expected_exception):
-    instance = {
-        'kafka_connect_str': KAFKA_CONNECT_STR,
-        'monitor_unlisted_consumer_groups': True,
-        'security_protocol': 'SASL_PLAINTEXT',
-        'sasl_mechanism': 'OAUTHBEARER',
-        'use_legacy_client': LEGACY_CLIENT,
-    }
-    instance.update(sasl_oauth_token_provider)
+def test_oauth_config(sasl_oauth_token_provider, check, expected_exception, kafka_instance):
+    kafka_instance.update(
+        {
+            'kafka_connect_str': KAFKA_CONNECT_STR,
+            'monitor_unlisted_consumer_groups': True,
+            'security_protocol': 'SASL_PLAINTEXT',
+            'sasl_mechanism': 'OAUTHBEARER',
+            'use_legacy_client': LEGACY_CLIENT,
```

Comment:
```suggestion
            'monitor_unlisted_consumer_groups': True,
            'security_protocol': 'SASL_PLAINTEXT',
            'sasl_mechanism': 'OAUTHBEARER',
```
nit: these are already in `kafka_instance`